### PR TITLE
types for paytmchecksum package

### PR DESCRIPTION
**Pull request by adarshsingh87**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`.

### CODE REVIEW

File: types/paytmchecksum/index.d.ts
Comment (codershiba) [inline types/paytmchecksum/index.d.ts:103]:
```suggestion
export = PaytmChecksum
```

**Comment by codershiba** [comment]
Types should be declared in namespace I guess?

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/paytmchecksum/paytmchecksum-tests.ts
1. types/paytmchecksum/index.d.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 6aa4bcae80d666b996ec1d121f7c3ac6f4e765eb
</p>

File: types/paytmchecksum/paytmchecksum-tests.ts
Comment (codershiba) [inline types/paytmchecksum/paytmchecksum-tests.ts:2]:
```suggestion
import PaytmChecksum = require("paytmchecksum");
```

**Review by ghost — APPROVED** [review]

**Comment by adarshsingh87** [comment]
I think @codershiba reviewed and approved it, is there any further review required from my side?

**Comment by codershiba** [comment]
> I think @codershiba reviewed and approved it, is there any further review required from my side?

It needs an approval from a Definitely Typed repo maintainer

**Comment by adarshsingh87** [comment]
oh! let's wait for it.

**Review by rbuckton — APPROVED** [review]

**Comment by adarshsingh87** [comment]
Ready to merge

**Pull request merged**
